apps/frontend/src/assets/css/variables.css

```diff
@@ -19,6 +19,7 @@
 	--light-gray: #e0e0e0;
 	--white: #ffffff;
 	--disable: #e0e0e0;
+	--switch-disable: #7c7c7c;
```

Comment:
I noticed that there are two variables with the same color `--text-secondary` and `--switch-disable`. Maybe its better to create one variable and call it something like `--medium-gray` and use it instead of `--text-secondary` and `--switch-disable`.

Reply:
`--text-secondary` is used only on task card, I created this variable in my last PR

Reply:
@marharita08  let's rename it to --medium-gray then